pkg/apis/v1alpha1/vpcnetworkconfiguration_types.go

```diff
@@ -21,14 +26,24 @@ const (
 type VPCNetworkConfigurationSpec struct {
 	// PolicyPath of Tier0 or Tier0 VRF gateway.
 	DefaultGatewayPath string `json:"defaultGatewayPath,omitempty"`
+
+	// VPCConnectivityProfile ID. This profile has configuration related to create VPC transit gateway attachment.
+	VPCConnectivityProfile string `json:"vpcConnectivityProfile,omitempty"`
+
+	// +kubebuilder:validation:Enum=SMALL;MEDIUM;LARGE;XLARGE
+	LbServiceSize string `json:"lbServiceSize,omitempty"`
```

Comment:
Could you add this to `VPCNetworkConfigInfo`?

Reply:
done

Reply:
No, the LB size should be only configured in VPCNetworkConfiguration CR

Reply:
`VPCNetworkConfigInfo` is an internal struct, and it is used to store VPC config.